src/routes/(console)/project-[region]-[project]/auth/+page.svelte

```diff
@@ -46,11 +59,47 @@
         }
     ]);
 
+    let selectedUsers: string[] = [];
+    let showDelete = false;
+    let deleting = false;
+
     async function userCreated(event: CustomEvent<Models.User<Record<string, unknown>>>) {
+        showCreateUser.set(false);
```

Comment:
not needed I think, the value is `bound` on the `Create` component below. Does it not work?